testsuite/features/support/constants.rb

```diff
@@ -275,12 +275,14 @@
   %w[
     sle-product-sles15-sp4-pool-x86_64
     sle-manager-tools15-pool-x86_64-sp4
+    sle-manager-tools15-pool-x86_64-sled-sp4
```

Comment:
Currently for Uyuni at least we partially sync SLED 15 SP3, if I understand properly the reposync kill condition, it will still be killed since we won't have any SLE15-SP3 product
We could either change the default dependency to SLED15-SP3 or update the synced channel to be SLED15-SP4

Reply:
> Currently for Uyuni at least we partially sync SLED 15 SP3, if I understand properly the reposync kill condition, it will still be killed since we won't have any SLE15-SP3 product We could either change the default dependency to SLED15-SP3 or update the synced channel to be SLED15-SP4

We do start a reposync of SLED 15 SP3 product in order to test the products wizard. As we don't really use this product, we can kill its reposyncs. However, two of them, the ones I add here, are shared with SLES 15 SP4 and should not be killed. Reason is that the tools are shared between all service packs.

I know, it's very convoluted.